codegen/test/rtc/include/rtc/tmp_dir.hpp

```diff
@@ -2,15 +2,18 @@
 #define GUARD_HOST_TEST_RTC_INCLUDE_RTC_TMP_DIR
 
 #include <string>
-#include <filesystem>
+//#include <filesystem>
```

Comment:
Why are we removing `std::filesystem` here?

Reply:
The problem is that filesystem is not supported on some of the older linux OSs, so this issue has been causing crashes in RHEL/CENTOS dockers and we can't promote CK to staging for some time now.

Reply:
Cant you just disable the tests for those older OSs?

Reply:
For older OSs, you can use filesystem TS instead. We have this header that defines the `fs` namespace to either `::std::filesystem` or `::std::experimental::filesystem`:

https://github.com/ROCm/AMDMIGraphX/blob/develop/src/include/migraphx/filesystem.hpp

Let me add a comment to the other PR.

Reply:
yes, we've created another branch where we're using that approach. i just verified that it builds correctly on all OSs.